docs/operator-manual/server-commands/argocd-application-controller.md

```diff
@@ -69,5 +69,11 @@ argocd-application-controller [flags]
       --token string                           Bearer token for authentication to the API server
       --user string                            The name of the kubeconfig user to use
       --username string                        Username for basic authentication to the API server
+      --wq-backoff-factor float                Set Workqueue Per Item Rate Limiter Backoff Factor, default is 1.5 (default 1.5)
+      --wq-basedelay-ns duration               Set Workqueue Per Item Rate Limiter Base Delay duration in nanoseconds, default 1000000 (1ms) (default 1ms)
+      --wq-bucket-qps int                      Set Workqueue Rate Limiter Bucket QPS, default 50 (default 50)
+      --wq-bucket-size int                     Set Workqueue Rate Limiter Bucket Size, default 500 (default 500)
+      --wq-cooldown-ns duration                Set Workqueue Per Item Rate Limiter Cooldown duration in ns, default 0(per item rate limiter disabled)
+      --wq-maxdelay-ns duration                Set Workqueue Per Item Rate Limiter Max Delay duration in nanoseconds, default 1000000000 (1s) (default 1s)
```

Comment:
imo we could really use a paragraph or two in docs explaining these options and how to experiment with them. If we'd like to solicit feedback from the community, we should make it super easy for them to understand how to tune these values so they can report back what they found to be effective.

Reply:
Where do you think would be a good place to put the doc? A separate file under operator manual or in some exisitng file?

Reply:
I think the best place would be along where we document other tuning variables:
https://github.com/argoproj/argo-cd/blob/ad4962352c7ac1aab66ec33dc2d81a150b80a927/docs/operator-manual/high_availability.md?plain=1#L22

I know the file is called "high availability," but there are already performance-oriented tuning variables documented and so it wouldn't be totally out of place.

Reply:
Makes sense, will add the docs for rate limiting there.

Reply:
Updated PR to include the docs PTAL.